src/utils/activityHistory.ts

```diff
@@ -188,7 +190,7 @@ const activityLogDynamoDBRequest = (
   ScanIndexForward: false, // Set to 'true' for ascending order
 });
 
-const getActivityLogEntry = async (
+export const getActivityLogEntry = async (
```

Comment:
very small thing, but shouldn't this be called `getActivityLogEntries`

Reply:
Yes, I agree. I'll update that in another PR later today.